doc/sphinx-guides/source/admin/external-tools.rst

```diff
@@ -94,6 +94,14 @@ File level preview tools allow the user to see a preview of the file contents wi
 
 When a file has a preview available, a preview icon will appear next to that file in the file listing on the dataset page. On the file page itself, the preview will appear in a Preview tab either immediately or once a guestbook has been filled in or terms, if any, have been agreed to.
 
+File Level Query Tools
+++++++++++++++++++++++++
+
+File level query tools allow the user to ask natural language queries of data table's contents without having to download it.
+
+When a file has a query tool available, a query icon will appear next to that file in the file listing on the dataset page. On the file page itself, the query tool will appear in a Query tab either immediately or once a guestbook has been filled in or terms, if any, have been agreed to.
+
+
```

Comment:
Something to consider: for explore tools we advertise them on the file landing page under Access File like in the screenshot below. Should we add "Query Options" and list "Ask Dataverse" there as well?

![Screen Shot 2023-07-26 at 2 38 56 PM](https://github.com/IQSS/dataverse/assets/21006/c54a6d32-9a62-477a-95c2-07451559b277)

Reply:
Query tools will now show here on the Access File button